contracts/EventEmitter.sol

```diff
@@ -16,6 +16,8 @@ contract EventEmitter {
     event LongUintBooleanString(uint256 uintValue, bool booleanValue, string stringValue);
     event Bytes(bytes value);
     event StringArray(string[] value);
+    event IndexedConflictingUint(uint256 indexed indexedValue, uint256 normalValue);
```

Comment:
Can you explain the choice of event argument names here as well as in `IndirectEventEmitter`?

Is there a specific reason why in this case `normalValue` is the second argument and is the first in the indirect contract?

Reply:
This is to more clearly show/test the conflict. However the names are irrelevant, the only important thing is the number of indexed parameters in the indirect emitter to be more than in the direct emitter.